ui/js/services/jobfilters.js

```diff
@@ -476,6 +477,11 @@ treeherder.factory('thJobFilters', [
                 _.isEqual(_toArray($location.search()[QS_CLASSIFIED_STATE]), ['unclassified']));
     };
 
+    var _isInProgressShown = function() {
+        return (_.contains(_getFiltersOrDefaults('filter-resultStatus'), 'pending') || 
```

Comment:
this can be accomplished with a single call to ``_getFiltersOrDefaults`` like this:

    return _.difference(['pending', 'running'], _getFiltersOrDefaults(RESULT_STATUS)).length === 0;

That's similar to what was being done before your patch, but it's nice to have this in a function that clearly states ``_isInProgressShown`` like you have here.